tsconfig.base.json

```diff
@@ -118,10 +118,7 @@
       "@skyux/lookup/testing": [
         "libs/components/lookup/testing/src/public-api.ts"
       ],
-      "@skyux/manifest": ["libs/components/manifest/src/index.ts"],
-      "@skyux/manifest/public-api.json": [
-        "libs/components/manifest/public-api.json"
-      ],
+      "@skyux/manifest": ["dist/libs/components/manifest"],
```

Comment:
Everything referencing the manifest should reference its `dist` folder since it will include the constructed JSON files.